dev/com.ibm.websphere.appserver.features/visibility/public/mpGraphQL-1.0/resources/l10n/com.ibm.websphere.appserver.mpGraphQL-1.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature provides support for the Eclipse MicroProfile GraphQL specification for enterprise Java.
```

Comment:
Recommend changing:
This feature provides support for the Eclipse MicroProfile GraphQL specification for enterprise Java.
to
This feature supports the Eclipse MicroProfile GraphQL specification for Enterprise Java.

Let me know if you have any questions or thoughts about these changes.

Reply:
Sounds good to me.  I'll change that now.  Thanks for the review!